Eastern-Research-Group/mywaterway
# Feature/bump code coverage

## Main Changes:
* Added tests for the add data widget.

## Steps To Test:
1. Run `npm run coverage`
2. Verify all of the tests pass
3. Verify the coverage value is above 70%.
